src/dss/main.py

```diff
@@ -30,5 +31,48 @@ def initialize_command(kubeconfig: str) -> None:
     initialize(lightkube_client=lightkube_client)
 
 
+CREATE_NOTEBOOK_IMAGE_HELP = """
+\b\nThe image used for the notebook server.  Suggested images:
+- kubeflownotebookswg/jupyter-scipy:v1.8.0 (default)
+- kubeflownotebookswg/jupyter-pytorch-full:v1.8.0
+- kubeflownotebookswg/jupyter-pytorch-cuda-full:v1.8.0
+- kubeflownotebookswg/jupyter-tensorflow-full:v1.8.0
+- kubeflownotebookswg/jupyter-tensorflow-cuda-full:v1.8.0
+"""
+
+
+@main.command(name="create-notebook")
+@click.argument(
+    "name",
+    required=True,
+)
+@click.option(
+    "--image",
+    default="kubeflownotebookswg/jupyter-scipy:v1.8.0",
+    help=CREATE_NOTEBOOK_IMAGE_HELP,
+)
+# FIXME: Remove the kubeconfig param from the create-notebook command (and any tests) after
+#  https://github.com/canonical/data-science-stack/issues/37
+@click.option(
+    "--kubeconfig",
+    help=f"Path to a Kubernetes config file. Defaults to the value of the KUBECONFIG environment variable, else to '{KUBECONFIG_DEFAULT}'.",  # noqa E501
+)
+def create_notebook_command(name: str, image: str, kubeconfig: str) -> None:
```

Comment:
I think we are missing functionality for `--gpu` and `--no-gpu` options which we mentioned in the doc here https://docs.google.com/document/d/1v0wU8TBb-ui6RcjGRbLwA6XFZnWbuiVb2TPmEgwKejY/edit#heading=h.wylzoqh1ph5w (check the full help message there is description of the options).

Reply:
to be addressed in #39